crates/revm/src/optimism/l1block.rs

```diff
@@ -192,21 +192,27 @@ impl L1BlockInfo {
     /// Introduced in isthmus. Prior to isthmus, the operator fee is always zero.
     pub fn operator_fee_charge(&self, input: &[u8], gas_limit: U256, spec_id: SpecId) -> U256 {
         // If the input is a deposit transaction or empty, the default value is zero.
-        if input.is_empty() || input.first() == Some(&0x7F) {
+        if input.is_empty() || input.first() == Some(&0x7E) {
```

Comment:
Why are we considering the operator fee to be zero based off of the input size? Isn't that not super important for what succinct is trying to charge for (compute units, rather than DA cost?)

Reply:
@yuwen01 Do you remember why you added `input.is_empty()`?